resources/assets/js/components/pages/home/NewSpot.vue

```diff
@@ -212,6 +235,8 @@
                 }).catch((error) => {
                     console.error(error);
                 });
+
+                this.apiHeaders = window.axios.defaults.headers.common;
```

Comment:
In order to get the Element file upload [component](https://element.eleme.io/#/en-US/component/upload#user-avatar-upload) working, the proper authentication headers need to be sent. Luckily, they're already created on page load for Axios, so we can just use those for this request.